frontend/src/app/features/work-packages/components/wp-relations/wp-relations.component.ts

```diff
@@ -115,7 +115,10 @@ export class WorkPackageRelationsComponent extends UntilDestroyedMixin implement
 
   private async updateFrontendData(event:CustomEvent) {
     if (event) {
-      const form = event.target as HTMLFormElement;
+      // A turbo:submit-end event *has* a `formSubmission` property, but I do not
+      // know how to avoid the eslint type warning. Please if you know, fix it.
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
FYI The Turbo typings aren't up-to-date or complete - I started working on a fix, but haven't revisited in a couple months https://github.com/opf/openproject/pull/17654